app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:  # Added return type annotation
+        self.name: str = name
+
+    def visit_cafe(self, visitor: dict) -> str:  # Added return type annotation
+        # Check if visitor is vaccinated
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"{visitor.get("name", "Visitor")} is not vaccinated"
+                f" and cannot enter {self.name}.")

```

Comment:
Code Style: You're using double quotes inside a string that's enclosed in double quotes. This will raise a `SyntaxError`. Use single quotes inside the string, like: `f'{visitor.get('name', 'Visitor')} is not vaccinated and cannot enter {self.name}.'`.